src/Pages/Overview/Overview.css

```
@@ -10,7 +10,7 @@
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: white; 
```

there seems to be a trailing space here after the semicolon, remove it
```suggestion
  color: white;
```

---

This wasn't resolved. Please fix